Sources/dep/Git.swift

```diff
@@ -43,9 +43,8 @@ class Git {
         }(self)
 
         var versions: [Version] {
-            //TODO separator is probably \r\n on Windows
             let out = (try? popen([Git.tool, "-C", root, "tag", "-l"])) ?? ""
-            let tags = out.characters.split("\n")
+            let tags = out.characters.split { $0 == "\n" || $0 == "\r\n" }.map(String.init)
```

Comment:
I am failing to see how this would world, since `$0` inside this split closure will be a single character, I don't see how it could ever contain two characters `\r\n`.